packages/muriel-typography/typography.scss

```
@@ -0,0 +1,70 @@
+.muriel-text-display-level-1 {
+  font-weight: regular;
```

```suggestion
  font-weight: 400;
```

Same for all instances of font-weight: ~regular~ normal;